.github/workflows/prepare-release.yml

```diff
@@ -101,8 +101,7 @@ jobs:
       #   - Run make prepare-release PREVIOUS_VERSION=0.52.0 RELEASE_CANDIDATE=0.53.0 MODSET=beta
       - name: Prepare release for core
         env:
-          BOT_GITHUB_TOKEN: ${{ secrets.OPENTELEMETRYBOT_GITHUB_TOKEN }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.OPENTELEMETRYBOT_GITHUB_TOKEN }}
```

Comment:
I don't understand this change

Reply:
Originally, the release script used the OTelBot token to create the PR (we can't use a normal CI token because workflows won't run). I previously changed it to use OTelBot token to create the PR, and the CI token to add the label. Since we're no longer adding a label, I'm just reverting it to how it used to be.